packages/flame_jenny/example/lib/components/button_row.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flame/components.dart';
+import 'package:jenny/jenny.dart';
+import 'package:jenny_example/components/dialogue_button.dart';
+
+class ButtonRow extends PositionComponent {
+  ButtonRow({required super.size}) : super(position: Vector2(0, 96));
+
+  void removeButtons() {
+    List<DialogueButton> buttonList = children.query<DialogueButton>();
+    if (buttonList.isNotEmpty) {
+      for (DialogueButton dialogueButton in buttonList) {
+        if (dialogueButton.parent != null) {
+          dialogueButton.removeFromParent();
+        }
+      }
+    }
+  }
+
+  void showNextButton(Function onNextButtonPressed) {
+    removeButtons();
+    DialogueButton nextButton = DialogueButton(
```

Comment:
```suggestion
    final nextButton = DialogueButton(
```
Once you add `flame_lint` it will come with some suggestions like this.